app/views/users/show.html.erb

```diff
@@ -14,7 +14,7 @@
       <div class="skills__header">
         <h2 class="skills__title text-2xl">Skills Progress</h2>
         <p class="skills__content">
-          The following courses should be taken in order.
+        <% current_user.path.courses.size > 1 ? 'The following courses should be taken in order' : null %>
```

Comment:
Was this tested locally? it's not valid erb unfortunately, you'll need to do something like this:

```ruby
<% if current_user.path.courses.size > 1 %>
  <p class="skills__content">
    The following courses should be taken in order
  </p>
<% end %>
```

Reply:
> it's not valid erb unfortunately

guilty!! i will bring that change right away, thanks :)

Reply:
@bappyasif cmon mate, you're wasting your own time and anyone who needs to review it when you're submitting code that you don't know will run or not or not. Please get the app running locally so you can test changes first.

Reply:
agreed!! but i told you before about that situation!! but sure will try to find a way, thanks @KevinMulhern